### PR TITLE
WIP Select

### PR DESCRIPTION
**Pull request by GuillaumeNury**

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

File: packages/ng/select2/src/lib/input/select-input.component.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@import '_definitions';
+@include selectInputStyle;
+@include selectClearerStyle;
+
+// TODO: Why ?
```
Comment (vvalentin-lucca) [inline packages/ng/select2/src/lib/input/select-input.component.scss:5]: stange

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1704/storybook

**Review by jeremie-lucca — APPROVED** [review]

File: packages/ng/simple-select/src/lib/input/select-input.component.ts [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+		this._containerElement.setAttribute('aria-labelledby', this.selectLabelId);
+		this._containerElement.setAttribute('role', 'listbox');
+		this._containerElement.id = `lu-select-overlay-container-${this.selectId}`;
+	}
+}
+
+@Injectable()
+class LuSelectOverlay extends Overlay {}
+
+class SelectPanelRef<T> extends LuSelectPanelRef<T> {
```
Comment (shajz) [inline packages/ng/simple-select/src/lib/input/select-input.component.ts:72]: This file does a lot of things, maybe we could split it ? A file for each class ?

File: packages/scss/src/components/index.scss [hunk 3]
```diff
@@ -44,7 +44,7 @@
 @forward 'button'; // 12 Ko
 @forward 'textfield'; // 13 Ko
 @forward 'util'; // 14 Ko
-@forward 'navSide'; // 15 Ko
+@forward 'navside'; // 15 Ko
```
Comment (shajz) [inline packages/scss/src/components/index.scss:47]: Sure about this? 🤔

Comment (lfaldalucca) [reply packages/scss/src/components/index.scss:47]: The actual file is lowercase. This line of code was breaking the build on my WSL setup.

File: packages/ng/simple-select/src/lib/input/select-input.component.ts (hunk 2 above)
Comment (lfaldalucca) [reply packages/ng/simple-select/src/lib/input/select-input.component.ts:72]: It is definitely planned.  The whole panel code is going to be replaced with a more modular approach.

Comment (lfaldalucca) [reply packages/ng/simple-select/src/lib/input/select-input.component.ts:72]: But that will be done in a subsequent pr.

**Review by shajz — APPROVED** [review]

**Pull request merged**